From c193444833a2b439e289f74c2a7db303f0e211ba Mon Sep 17 00:00:00 2001
From: Wang Haoran <b50027@freescale.com>
Date: Fri, 5 Sep 2014 10:37:32 +0800
Subject: [PATCH 04/22] ENGR00326131-4 Porting Realtek RTL8821AS wifi card

Use the Realtek patches for Athores's wpa_supplicant.
That will improve the performance and fix some p2p
issues for AR6223 wifi card.

Signed-off-by: Wang Haoran <b50027@freescale.com>
---
 wpa_supplicant/Android.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/wpa_supplicant/Android.mk b/wpa_supplicant/Android.mk
index ba8cb79..560c982 100644
--- a/wpa_supplicant/Android.mk
+++ b/wpa_supplicant/Android.mk
@@ -1580,6 +1580,7 @@ LOCAL_STATIC_LIBRARIES += libnl_2
 endif
 endif
 LOCAL_CFLAGS := $(L_CFLAGS)
+LOCAL_CFLAGS += -DREALTEK_WIFI_VENDOR
 LOCAL_SRC_FILES := $(OBJS)
 LOCAL_C_INCLUDES := $(INCLUDES)
 include $(BUILD_EXECUTABLE)
-- 
1.8.0

